src/components/Reports/StaffExpenseReport/SettingsDialog/SettingsDialog.test.tsx

```diff
@@ -325,27 +402,35 @@ describe('SettingsDialog', () => {
     expect(onClose).toHaveBeenCalledWith(originalFilters);
   });
 
-  it('should handle undefined selectedFilters', () => {
-    const { getByLabelText, getByText } = render(
+  it('should handle undefined selectedFilters and undefined categories', async () => {
+    const { getByLabelText, getByRole, rerender } = render(
       <TestComponent {...defaultProps} selectedFilters={undefined} />,
     );
 
-    expect(getByText('Report Settings')).toBeInTheDocument();
+    expect(
+      getByRole('heading', { name: 'Report Settings' }),
+    ).toBeInTheDocument();
+
+    await waitFor(() => {
```

Comment:
You can remove this `.toBeInTheDocument()` and just assert that `findByLabelText('Benefits')` is not checked.